tests/README.md

```diff
@@ -1,52 +1,54 @@
 # Apps Tests
 
 The test suite is implemented using [`bats`](https://github.com/bats-core/bats-core) and [`cypress`](https://github.com/cypress-io/cypress), with unit, regression, integration, and end-to-end tests under their own respective directory.
-Tests implemented with `cypress` can and will be integrated into the `bats` test suite using generators.
+
+Generators are employed to generate `bats` tests from `cypress` and `gotmpl` tests to integrate it into the rest of test suites.
```

Comment:
This was a bit hard to parse. What about something like this?
```suggestion
For each `cypress` test a `bats` test is generated that runs the `cypress` test and verifies the result.
Some tests are also generated from `gotmpl` templates.
```

Reply:
On the other hand there is already good descriptions of the different tests that you can write and the code generation further down so maybe this can be removed?